src/USER-CONP/fix_charge_update.h

```diff
@@ -44,9 +44,11 @@ class FixChargeUpdate : public Fix {
   std::vector<std::vector<double> > elastance;
   bigint ngroup;
   std::vector<tagint> taglist, taglist_bygroup, group_idx;
+  std::vector<int> tag_to_iele;
```

Comment:
Was macht das tag_to_iele genau bzw. was steht darin?

Reply:
Index in mpos für jedes tag. Dadurch brauchen wir keine verschachtelte Schleife mehr um mpos zu aktualisieren.